dashboarding/conversion_metrics.py

```diff
@@ -36,10 +36,12 @@ def get_net_worth_as_million_dollars(worth: int) -> str:
     """ Returns net worth as N millions (i.e 180 *10^6 -> 180M) """
     return f"${round(worth/10**6)}M"
 
+
 def get_most_recent_flight_info(owner:DataFrame, flights: DataFrame, aircraft:DataFrame) -> dict:
```

Comment:
Minor issue but spacing between the variable and the type hints is inconsistent here.